# The angle of the hip, with moves the leg in the plane of the body.
int32 hip_angle
# The angle of the knee, which rotates perpendicular to the hip ( axis points about the robot  in rest )
int32 knee_angle
# The angle of the ankle, the which rotates parallel to the knee,
int32 ankle_angle
